packages/diagrams/frontend/sirius-components-diagrams/src/renderer/node/FreeFormNode.tsx

```diff
@@ -77,15 +77,20 @@ export const FreeFormNode = memo(({ data, id, selected, dragging }: NodeProps<Fr
 
   const theme = useTheme();
   const { onDrop, onDragOver } = useDrop();
-  const { style: connectionFeedbackStyle } = useConnectorNodeStyle(id, data.nodeDescription.id);
-  const { style: dropFeedbackStyle } = useDropNodeStyle(data.isDropNodeTarget, data.isDropNodeCandidate, dragging);
 
-  const rotation = computeBorderRotation(data);
+  const rotation = useMemo(() => computeBorderRotation(data), [data.isBorderNode, data.positionDependentRotation]);
   let imageURL: string | undefined = undefined;
```

Comment:
Should depend on `data.borderNodePosition` ?

Reply:
yes, thanks